knative-serving/config-applications/startup-probes-for-serving.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="startup-probes-for-serving"]
+= Startup probes
+:context: startup-probes-for-serving
+
+toc::[]
```

Comment:
Unrelated to this PR (because it's fine here), but I was poking around and saw that quite a few other assemblies seem to be missing the right-hand TOC. I assume that they're missing this `toc::[]` designation. Just noting as something to look into in a future update.